src/sql/src/plan/statement.rs

```diff
@@ -718,6 +718,11 @@ impl<'a> StatementContext<'a> {
         Ok(())
     }
 
+    /// Returns an error if the named `FeatureFlag` is not set to `on`.
```

Comment:
nit: copypasta

Larger comment: if you're only using a chain of `std` functions, I prefer not wrapping those in add'l functions hanging off of other structs, i.e. just expecting folks to call `self.require_feature_flag(flag).is_ok()` is 👌

Reply:
sounds good!